recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+
+build:
+    number: 1
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython >=0.15
+    - numpy >=1.7
+    
+  run:
+    - python >=2.7
+    - cython >=0.15
+    - numpy >=1.7
+    - scipy >=0.14
+    - nose
+    - matplotib
+
+test:
+  imports:
+    - qutip
+  commands:
+    - from qutip import *
```

Comment:
Hmmm. Not sure this will work. The commands are CLI commands. In all honesty, this line is redundant due to the imports test above.